.github/workflows/master.yml

```diff
@@ -40,15 +40,13 @@ jobs:
           build_extra_args: "--compress=true"
       - uses: engineerd/setup-kind@v0.4.0
       - name: End2End
-        env:
-          DOCKER_USERNAME: ${{ github.actor }}
-          DOCKER_PASSWORD: "${{ secrets.GITHUB_TOKEN }}"
         run: |
+          docker build -t lyft/flytepropeller:${{ github.sha }} .
```

Comment:
hmmm this is something I would like to dig deeper in. My understanding is that steps will run on the same machine. This means that the previous step that just build the docker image should still have the image on the docker cache. Can we just use it?

Reply:
yes i used it before also and for 100% confirmation i can add imagePullPolicy to `Never`

Reply:
My point is, can we get rid of this line? because the previous step will ensure this image does exist. right?